src/qt/guiutil.h

```diff
@@ -224,6 +225,9 @@ namespace GUIUtil
     /* Convert OS specific boost path to QString through UTF-8 */
     QString boostPathToQString(const fs::path &path);
 
+    /* Convert ConnectionType enum to QString */
```

Comment:
571b0469fa952be98874d1afd772a7fa708dd83f

This comment is not Doxygen-compatible.

Reply:
thanks! fixed